crates/core/src/kernel/models/schema.rs

```diff
@@ -554,6 +556,7 @@ impl Display for PrimitiveType {
             PrimitiveType::Binary => write!(f, "binary"),
             PrimitiveType::Date => write!(f, "date"),
             PrimitiveType::Timestamp => write!(f, "timestamp"),
+            PrimitiveType::TimestampNtz => write!(f, "timestampNTZ"),
```

Comment:
nit: NTZ -> Ntz

Reply:
I am not sure anymore for what exactly, but I had to do this on purpose

Reply:
Was not needed after all xD